src/nexmo/__init__.py

```diff
@@ -83,6 +83,7 @@ def __init__(
         self.signature_secret = signature_secret or os.environ.get(
             "NEXMO_SIGNATURE_SECRET", None
         )
+
```

Comment:
extra whitespace

Reply:
I'm OK with extra whitespace and wouldn't block merging because of it.